.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "editor.formatOnSave": true,
+    "editor.formatOnPaste": true,
+    "[javascript]": {
```

Comment:
@lascenify in case you haven't come across this file before, the settings here apply to everyone using VSCode with this project. I'm setting the Prettier formatter to be the default so that we all format using the same conventions.